### DISCUSSION TITLE
How to use the tools without prependeing coreutils to them?

### DISCUSSION THREAD

**Discussion by LinArcX** [uutils/coreutils#5846] [Q&A] [answered]
For example i prefer to run:
`ls`
instead of:
`coreutils.exe ls`

How can i do that?

**Comment by sylvestre** [comment]
See the doc ["make"](https://github.com/uutils/coreutils#gnu-make)

**Reply by LinArcX** [reply]
im on windows, not sure if make works there.

**Comment by tertsdiepraam** [comment, marked as answer]
To expand on the other answer, you can set up aliases in your shell or create symlinks with different names to the coreutils binary. If you symlink `ls` to `coreutils` and call it, it will recognize that you called it as `ls` and act like `ls`.
